.expeditor/scripts/build_mac_component.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/local/bin/bash
+
+set -euo pipefail
+
+source .expeditor/scripts/shared.sh
+
+# Get secrets!
+eval "$(vault-util fetch-secret-env)"
+
+export HAB_AUTH_TOKEN="${ACCEPTANCE_HAB_AUTH_TOKEN}"
+export HAB_BLDR_URL="${ACCEPTANCE_HAB_BLDR_URL}"
+
+########################################################################
+
+# `component` should be the subdirectory name in `components` where a
+# particular component code resides.
+#
+# e.g. `hab` for `core/hab`, `plan-build` for `core/hab-plan-build`,
+# etc.
+component=${1:?You must specify a component value}
+
+channel=$(get_release_channel)
+
+echo "--- Channel: $channel - bldr url: $HAB_BLDR_URL"
+
+echo "--- Installing buildkite agent"
+brew tap buildkite/buildkite
+brew install --token="$BUILDKITE_AGENT_ACCESS_TOKEN" buildkite-agent
+
+echo "--- Installing mac bootstrap package"
+bootstrap_package_version="1.0.7"
```

Comment:
Do we want this hard-coded rather than a "latest" package?   Future iterations on the bootstrap package would require us to remember to bump this pipeline.  

Not a blocker ( I assume that would require work on the mac omnibus pipeline ), just a thought.

Reply:
The future of this will be in a file, similar to the `rust-version` files so we can set up a subscription for when the mac-bootstrapper gets updated. So yes!